src/components/ItemsList.js

```diff
@@ -1,49 +1,45 @@
 import React, { useState } from 'react';
-import firebase from '../lib/firebase';
+import { shoppingListCollection } from '../lib/firebase';
 import Nav from './Nav';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 import { useHistory } from 'react-router-dom';
 import calculateEstimate from '../lib/estimates';
 import SearchBar from './SearchBar';
 import {
-  getShoppingItemBackgroundStyles,
   getItemDescription,
   sortShoppingList,
-} from './sortingFunctions';
+  getShoppingItemBackgroundStyles,
+  wasItemPurchasedWithinLastOneDay,
+  getDaysBetweenCurrentAndPreviousPurchase,
+} from '../utils/utility-functions';
 import AddItemButton from './AddItemButton';
 import { ReactComponent as TrashBin } from '../img/trash-alt-regular.svg';
 import { ReactComponent as HomeIcon } from '../img/home-solid.svg';
 import spinner from '../img/spinner-3.gif';
+import ConfirmDeleteModal from './ConfirmDeleteModal';
 import '../index.css';
```

Comment:
I think there is no need for importing index.css here since it is in the index.js